src/common/drain/drain.js

```diff
@@ -6,33 +6,66 @@
 import { ee } from '../event-emitter/contextual-ee'
 import { mapOwn } from '../util/map-own'
 import { registerHandler as defaultRegister } from '../event-emitter/register-handler'
+import { featurePriority } from '../../loaders/features/features'
 
+const registry = {}
 
-// calls will need to update to call this more directly so we can explicitly pass in the ee and handler
-export function drain(agentIdentifier, group = 'feature') {
-  const baseEE = agentIdentifier ? ee.get(agentIdentifier) : ee
-  const handlers = defaultRegister.handlers
-  if (!baseEE.backlog || !handlers) return
+export function registerDrain(agentIdentifier, group) {
```

Comment:
Each feature `inst` will call `registerDrain` to help the drain module predict what all needs draining later on